backend/de.metas.manufacturing/src/main/java/org/eevolution/api/PPOrderCreateRequest.java

```diff
@@ -99,8 +99,6 @@
 			@Nullable final BPartnerId customerId,
 			@Nullable final ProjectId projectId,
 			//
-			final boolean pickingOrder,
```

Comment:
i think we shouldn't remove it (bc of the follow-up-issue)

Reply:
the field was actually present only on the builder (probably forgotten after certain refactor work ).. so no functional changes here, only clean up...
the `pickingOrder` flag from `PP_Order` ends up there directly from pp_product_planning, see: `org/eevolution/api/impl/CreateOrderCommand.java:158` 

![image](https://user-images.githubusercontent.com/83770997/143221364-9962ffd5-16e2-45d9-b1aa-fff86d86dd2d.png)